examples/classifier/CL4_apply_DTW_based_kmeans_clustering.py

```diff
@@ -0,0 +1,87 @@
+# Standard library imports
+import sys
+new_path = sys.path[0][:-len("/examples/timeseries")]
+sys.path.append(new_path)
```

Comment:
Not sure I understand why setting path is needed here. 

Anyway, there is also another library called pathlib which helps you set the directory or filepath, E.g. 

```
from pathlib import Path
directory = Path(__file__).parent   # here directory = parent folder of current module file
```

Reply:
Without setting this relative path, I was not able to call scifin packages. I must say it is not a very clean solution, but it works. I hope it works on other machines as well.
Thanks for pathlib recommendation, I will check it out!

Reply:
I removed the imports in one of the scripts and it run fine provided that you are in the poetry environment meaning that either you do:
```
poetry run python examples/....
```
or you create the env first:
```
poetry shell
``` 
and then in the shell you do:
```
python examples....
```

Reply:
It does not seem to work on my side.

Reply:
Testing under poetry env also seem pretty slow...

Reply:
How do you know it is slow if it doen't work?

Also, have you run
```
poetry install
```
?
You should see poetry installing the scifin package locally.

Reply:
Indeed. That was the missing piece. I can remove the path trick then.